services/spar/test-integration/Test/Spar/Scim/UserSpec.hs

```diff
@@ -463,13 +466,31 @@ specUpdateUser = describe "PUT /Users/:id" $ do
         it "fails to update user with 404" testUserUpdateFailsWithNotFoundIfOutsideTeam
     context "scim_user has no entry with this id" $ do
         it "fails" $ pending
+    it "does not update if nothing changed" $ testSameUpdateNoChange
     context "brig user is updated" $ do
+        -- TODO(arianvp): This will be fixed by
```

Comment:
A note here; we have a field in `brig` named: `managed_by`, and one of the values is `scim`. Ideally, users that are managed by `scim` should not be able to update fields since it makes sense for the IdP to be the source of truth.